src/main/kotlin/no/fdk/dataset_catalog/service/DatasetService.kt

```diff
@@ -45,7 +45,7 @@ class DatasetService(
             .copy(
                 id = datasetId,
                 catalogId = catalogId,
-                lastModified = LocalDateTime.now(),
+                lastModified = ZonedDateTime.now(),
```

Comment:
Hva er gevinsten av å endre til `ZonedDateTime`?

Reply:
Front-end og data som er lagret i gamle dataset-catalog bruker Date objekter med tidssone. LocalDateTime har ingen tidssone, som førte til at sommertid og tidssone kunne endre tidspunkt med opptil to timer. Felt som bare er lagret som datoer - som `modified` - ligger inne som yyyy-MM-ddT00:00:00, som derfor rullet tilbake en dag ved minus 1 eller 2 timer